src/smpdflib/plots.py

```diff
@@ -132,6 +134,51 @@ def plot_pdfs(pdfsets, Q, base_pdf = None, flavors=None, photon=False):
 
         yield (fl,),fig
 
+def plot_pdfcorr(pdfsets, Q, base_pdf=None, flavors=None, photon=False):
+
+    if flavors is None:
+        flavors = PDF.make_flavors(photon=photon)
+
+    nflavors = len(flavors)
+    xgrid = PDF.make_xgrid()
+
+    resultlists = []
+    for pdf in pdfsets:
+        mean, replicas = pdf.grid_values(Q, xgrid, flavors)
```

Comment:
This is correct for MC and symhessian sets, but would give wrong results for hessian (not to mention the horrible +as thing and so on). We should either implement it or forbid also Hessian sets for this action (with a check function). If we are to handle it, it should be trough make_pdf_result, which returns objects that should know how to handle different uncertainty conventions.

Also, this problem is likely to exist for other tools using correlations.